conf/ledgersmb-apache-2.4.template

```diff
@@ -0,0 +1,60 @@
+# This file must be edited to substitute the current path with WORKING_DIR
+
```

Comment:
Ideally, since the file now declares a virtual host, we should explain how to integrate the file into the apache documentation, either here or by referring people to some documentation about virtual host configuration.

Another option would be to explain to people how to restrict all these commands to a single vhost by including the config into a vhost configuration with the INCLUDE command.